### PR TITLE
smallTyposSweep.treatments.md

### PR DESCRIPTION
**Pull request by yemarshall**
As I was (re-)reading the manuscript version from ~May 1st I found a couple of typos, so I'm suggesting/inserting these typo corrections together and separately from deeper changes!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.
N/A

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -561,7 +561,7 @@ Interferons (IFNs) are a family of cytokines crucial to activate the first (inna
 Interferons are classified into three categories based on their receptor specificity: type I, II and III [@doi:10.1128/mmbr.05015-11].
 Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation [@doi:10.1016/j.jcv.2003.11.013].
 Among these IFNs, IFN- 𝛽 was already found to strongly inhibit the replication of other corona viruses, such as SARS-CoV, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context [@doi:10.1016/j.jcv.2003.11.013].
-There are evidences that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- 𝛽.
+There is evidence that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- 𝛽.
 For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024].
```
Comment (rando2) [inline content/20.treatments.md:565]:
```suggestion
For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024].
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -561,7 +561,7 @@ Interferons (IFNs) are a family of cytokines crucial to activate the first (inna
...
 
 On March 18 2020 Synairgen plc has received approval to start a phase II trial for SNG001, an IFN- 𝛽-1a formulation to be delivered to lungs via inhalation.
```
Comment (rando2) [inline content/20.treatments.md:567]:
```suggestion
On March 18 2020 Synairgen plc has received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to lungs via inhalation.
```

File: content/20.treatments.md [hunk 3]
```diff
@@ -643,11 +643,11 @@ Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance li
 
 mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines.
 In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007].
```
Comment (rando2) [inline content/20.treatments.md:645]:
```suggestion
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007].
```

File: content/20.treatments.md [hunk 4]
```diff
@@ -32,26 +32,26 @@ In a healthy situation the lung respiratory epithelium together with alveolar ma
...
 
-Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
+Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
 Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [@url:https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741].
 Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April.
 The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyze its efficiency/safety (Biopharma-reporter.com).
```
[the remark this answers is not in the document]
Comment (cgreene) [reply content/20.treatments.md:53]: Does this need to link to a specific location at this website?

[the remark this answers is not in the document]
Comment (cgreene) [reply content/20.treatments.md:56]: Found https://www.biopharma-reporter.com/Article/2020/03/19/Roche-enters-Phase-III-for-COVID-19-treatment which I think is it & edited the suggestion to that.

**Pull request merged**